tox.ini

```diff
@@ -1,13 +1,22 @@
 [tox]
 skipsdist = true
-envlist = discopane-ui-tests, flake8
+envlist = discopane-ui-tests, flake8, ui-smoke-tests
```

Comment:
How about just calling this `smoke-tests` and naming the directory `tests/smoke`? I'd rather not keep using the term `ui` because everything about `addons-frontend` is already a UI. It's easy to confuse "ui" tests with those of our general purpose [ui components](https://github.com/mozilla/addons-frontend/tree/master/tests/unit/ui/components).